buildkitd/oom-adjust.sh.template

```diff
@@ -8,22 +8,33 @@ fi
 
 for PID in $(pidof buildkit-runc)
 do
-    PID_NAME=$(cat /proc/"$PID"/cmdline)
+    if [ ! -d /proc/"$PID" ]; then
+        ! "${BUILDKIT_DEBUG}" || echo "$(date) | $PID was missing, ignoring" >> /var/log/oom_adj
+        continue
+    fi
+
+    PID_NAME=$(cat /proc/"$PID"/cmdline || echo "unknown")
```

Comment:
Could this be simplified to do the `if` afterward, and instead check for `$PID_NAME == "unknown"`? It would be more atomic this way - there's only one interaction with `/proc/$PID` instead of two.

Reply:
Eh I went with it